units/patch_intent.go

```diff
@@ -159,6 +159,16 @@ func (j *patchIntentProcessor) Run(ctx context.Context) {
 }
 
 func (j *patchIntentProcessor) finishPatch(ctx context.Context, patchDoc *patch.Patch) error {
+	// Add owner and repo for child patches
+	if patchDoc.Author == evergreen.ParentPatchUser {
```

Comment:
Should we maybe use p.IsParent() here?

Reply:
is think you meant isChild but yes! i think thats way better